### PR TITLE
update webApp files

### PR DESCRIPTION
**Pull request by francisjodi**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,19 +1,275 @@
...
+                            <span>60%</span>
+                        </div>
+                        <div class="line python"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- teams section start
```
Comment (sammcd) [inline portfolio/src/main/webapp/index.html:166]: May want to consider deleting the commented out code if it is no longer useful.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,19 +1,275 @@
...
+                    </form>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- footer section start -->
+    <footer>
+        By: Jodi-Ann Francis 
+        <!-- <span>Created By <a href="https://www.codingnepalweb.com">CodingNepal</a> | <span class="far fa-copyright"></span> 2020 All rights reserved.</span> -->
```
Comment (sammcd) [inline portfolio/src/main/webapp/index.html:270]: Re-using code is great, but is there a license for the code you could link to? Does this require any attribution requirements where they ask to leave this sort of comment?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,19 +1,275 @@
...
+                </div>
+                <div class="card">
+                    <div class="box">
+                        <i class="fas fa-utensils"></i>
+                        <div class="text">Recipe App</div>
+                        <p>This recipe webapp runs on Google App Engine and uses the Recipe Puppy API to allow you to search for recipes and display search results
+                        </p>
+                    </div>
+                </div>
+                <!-- <div class="card">
```
Comment (sammcd) [inline portfolio/src/main/webapp/index.html:103]: Can possibly delete the commented out code.

**Review by sammcd — CHANGES_REQUESTED** [review]
Looks good. My only request is that does the coding nepal code used require any attribution or have any limitations on their license?

**Review by mikeshen — COMMENTED** [review]
Hey Jodi, I saw you added your week 2 changes! Thanks very much. However, can we actually do these changes as a separate pull request? Let's get this PR into a state with just the week 1 changes, and then I can approve.

**Comment by mikeshen** [comment]
[Github - About pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) has a lot of good info on the basic workflow of a Pull Request. Do message me if you can't figure out how to take off commits from a PR. I an walk you through the process.

**Comment by mikeshen** [comment]
Approved, week 1 changes look great!

**Review by mikeshen — APPROVED** [review]
Approved

**Pull request merged**